.travis.yml

```diff
@@ -16,7 +16,6 @@ matrix:
   - os: linux
     language: python
     python: 3.6
```

Comment:
I _think_ we can remove this job altogether as previously it was just running `flake8` and `black` (for an example build see https://travis-ci.org/github/dask/distributed/jobs/727483894)